test/config_test.js

```diff
@@ -179,6 +179,19 @@ describe('ConnectionConfig', () => {
         config.isBareAddresses('nsqdTCPAddresses', ['localhost'])
       check.should.throw()
     })
+
+    it('should validate ipv6 addresses correctly', () => {
+      const check = () =>
+        config.isBareAddresses('nsqdTCPAddresses', ['[::1]:4150', '[::]:4150'])
+      check.should.not.throw()
+    });
+
+    // TODO check invalid situations
+    it('should not validate non-numeric port of ipv6 addresses', () => {
+      const check = () =>
+        config.isBareAddresses('nsqdTCPAddresses', ['[::1]'])
```

Comment:
This doesn't look like it's testing the situation described in the test description, right?

Reply:
Remove the TODO and rename the description. Not sure if I have understood correctly (sry for my poor english).